hierarc/LensPosterior/kin_constraints_composite.py

```diff
@@ -152,16 +155,23 @@ def __init__(
             log_m2l_scaling=log_m2l_scaling,
         )
 
-        if self._check_arrays(kappa_s_array, r_s_angle_array):
-            self._kappa_s_array = kappa_s_array
+        if self._check_arrays(alpha_Rs_array, r_s_angle_array):
+            self._halo_normalization_array = alpha_Rs_array
+            self._alpha_Rs_or_kappa_s_bool = True
```

Comment:
Sorry that I didn't explicitly say about this the first time. `self._alpha_Rs_or_kappa_s_bool` is not particularly a clear naming for what it means when it's `True`. I suggest, `self._is_normalization_alpha_Rs` would be clearer.

Reply:
Done!  Thanks for reviewing the changes!